extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js

```diff
@@ -115,51 +115,46 @@ $(function () {
 						// not exists, add it to cachedVals and remove previous errors
 						cachedVals.push(val);
 
-						$this
-							.removeClass('error')
-							.popover('destroy');
-
+						$this.removeError();
 						return true;
 					} else if (optionCheckEmpty || val !== '') {
 						// if it exists and it's not empty it's duplication
-						$this
-							.addClass('error')
-							.popover('destroy')
-							.popover({
-								content: duplicateError
-							});
+						$this.addError(duplicateError);
 					}
 				});
 			},
+			checkImages = function () {
+				$ul.find('.image.error').removeError();
+
+				// find all images for items and sections except Featured Section and...
+				$ul.find('.section:not(.featured), .item')
+					.find('.image[data-id=0], .image:not([data-id])')
+					.addError(imageMissingError);
+				// ...except Optional Section
+				var $lastSection = $ul.find('.section').last().find('.section-input');
+				if ($lastSection.val().length === 0) {
+					$lastSection.parent().find('.image').removeError();
+				}
+			},
 			checkForm = function () {
 				$save.removeClass();
 
-				checkInputs($ul.find('.section-input'), ['required', 'limit']);
+				checkInputs($ul.find('.section-input'), ['limit', 'checkEmpty']);
```

Comment:
It's fix for bug that we've found - it should possible to add optional section (without a name), but only once - those flags were wrong since last code refactor.